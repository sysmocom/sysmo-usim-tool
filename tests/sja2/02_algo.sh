#!/bin/sh
. ./test-data

ALGOS="1:1 3:1 4:1 5:1 15:1 1:4 3:15 2:2 1:3 1:2:3 1:4:5 5:4:15 "
ALGOS=$ALGOS"MILENAGE:MILENAGE COMP128v1:MILENAGE COMP128v3:MILENAGE COMP128v2:MILENAGE SHA1-AKA:XOR COMP128v1:MILENAGE COMP128v1:MILENAGE:SHA1-AKA COMP128v2:COMP128v1:COMP128v3"
for algo in $ALGOS; do
	$TOOL -a $ADMPIN -T $algo
	$TOOL -a $ADMPIN -t
done
