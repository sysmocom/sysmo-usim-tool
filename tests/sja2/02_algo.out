sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 1=COMP128v1
   3g: 1=COMP128v1
   4g5g: 1=COMP128v1
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 1=COMP128v1
   3g: 1=COMP128v1
   4g5g: 1=COMP128v1

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 3=COMP128v3
   3g: 1=COMP128v1
   4g5g: 1=COMP128v1
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 3=COMP128v3
   3g: 1=COMP128v1
   4g5g: 1=COMP128v1

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 4=MILENAGE
   3g: 1=COMP128v1
   4g5g: 1=COMP128v1
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 4=MILENAGE
   3g: 1=COMP128v1
   4g5g: 1=COMP128v1

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 5=SHA1-AKA
   3g: 1=COMP128v1
   4g5g: 1=COMP128v1
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 5=SHA1-AKA
   3g: 1=COMP128v1
   4g5g: 1=COMP128v1

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 15=XOR
   3g: 1=COMP128v1
   4g5g: 1=COMP128v1
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 15=XOR
   3g: 1=COMP128v1
   4g5g: 1=COMP128v1

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 1=COMP128v1
   3g: 4=MILENAGE
   4g5g: 4=MILENAGE
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 1=COMP128v1
   3g: 4=MILENAGE
   4g5g: 4=MILENAGE

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 3=COMP128v3
   3g: 15=XOR
   4g5g: 15=XOR
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 3=COMP128v3
   3g: 15=XOR
   4g5g: 15=XOR

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 2=COMP128v2
   3g: 2=COMP128v2
   4g5g: 2=COMP128v2
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 2=COMP128v2
   3g: 2=COMP128v2
   4g5g: 2=COMP128v2

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 1=COMP128v1
   3g: 3=COMP128v3
   4g5g: 3=COMP128v3
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 1=COMP128v1
   3g: 3=COMP128v3
   4g5g: 3=COMP128v3

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 1=COMP128v1
   3g: 2=COMP128v2
   4g5g: 3=COMP128v3
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 1=COMP128v1
   3g: 2=COMP128v2
   4g5g: 2=COMP128v3

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 1=COMP128v1
   3g: 4=MILENAGE
   4g5g: 5=SHA1-AKA
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 1=COMP128v1
   3g: 4=MILENAGE
   4g5g: 4=SHA1-AKA

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 5=SHA1-AKA
   3g: 4=MILENAGE
   4g5g: 15=XOR
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 5=SHA1-AKA
   3g: 4=MILENAGE
   4g5g: 4=XOR

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 4=MILENAGE
   3g: 4=MILENAGE
   4g5g: 4=MILENAGE
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 4=MILENAGE
   3g: 4=MILENAGE
   4g5g: 4=MILENAGE

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 1=COMP128v1
   3g: 4=MILENAGE
   4g5g: 4=MILENAGE
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 1=COMP128v1
   3g: 4=MILENAGE
   4g5g: 4=MILENAGE

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 3=COMP128v3
   3g: 4=MILENAGE
   4g5g: 4=MILENAGE
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 3=COMP128v3
   3g: 4=MILENAGE
   4g5g: 4=MILENAGE

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 2=COMP128v2
   3g: 4=MILENAGE
   4g5g: 4=MILENAGE
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 2=COMP128v2
   3g: 4=MILENAGE
   4g5g: 4=MILENAGE

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 5=SHA1-AKA
   3g: 15=XOR
   4g5g: 15=XOR
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 5=SHA1-AKA
   3g: 15=XOR
   4g5g: 15=XOR

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 1=COMP128v1
   3g: 4=MILENAGE
   4g5g: 4=MILENAGE
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 1=COMP128v1
   3g: 4=MILENAGE
   4g5g: 4=MILENAGE

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 1=COMP128v1
   3g: 4=MILENAGE
   4g5g: 5=SHA1-AKA
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 1=COMP128v1
   3g: 4=MILENAGE
   4g5g: 4=SHA1-AKA

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initalizing...
 * New algorithm setting:
   2g: 2=COMP128v2
   3g: 1=COMP128v1
   4g5g: 3=COMP128v3
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Authentication parameters...
 * Initalizing...
 * Reading...
 * Current algorithm setting:
   2g: 2=COMP128v2
   3g: 1=COMP128v1
   4g5g: 1=COMP128v3

Done!
